content/millennium/dstu2/infrastructure/document-reference.md

```diff
@@ -108,8 +115,11 @@ List an individual DocumentReference by its id:
 
 _Implementation Notes_
 
-* Contents of the document are found by following the Attachment URL. See more information on the [Binary resource] to determine what Authorization scopes are required, and how to set the Accept header when downloading document contents.
-* If no mappings are available for LOINC codes when returning DocumentReference.type codings, an unknown data absent reason will be returned in place of the LOINC codes. This is to follow the [Argonaut profile's](http://www.fhir.org/guides/argonaut/r2/StructureDefinition-argo-documentreference.html) required binding for type. When available, Proprietary codings will be returned in addition to the data absent reason.
+- Contents of the document are found by following the Attachment URL. 
+  - See more information on the [`Binary`] resource to determine what Authorization scopes are required, and how to set the `Accept` header when downloading document contents.
+- If no mappings are available for LOINC codes when returning `type` codings, an unknown data absent reason will be returned in place of the LOINC codes. 
+  - This is to follow the [Argonaut profile's](http://www.fhir.org/guides/argonaut/r2/StructureDefinition-argo-documentreference.html) required binding for type. 
+  - When available, Proprietary codings will be returned in addition to the data absent reason.
```

Comment:
I thought it was weird that we had this call-out there, since we don't reference stuff like this in almost any other spec page of ours.

I don't think there's anything wrong with having the reference out to Binary since that's still appropriate, but I think the lines about the `type` can be droppd.  The same bindings apply for both searching and retrieving functions - you just either are searching with parameters, or pulling up a specific ID.